src/vivarium_cluster_tools/psimulate/distributed_worker.py

```diff
@@ -98,66 +98,42 @@ def fork_work_horse(self, job, queue):
             self.procline("Forked {0} at {1}".format(child_pid, time()))
 
 
-def worker(parameters: Mapping):
+def worker(job_parameters: dict):
     node = f"{os.environ['SGE_CLUSTER_NAME']}:{os.environ['HOSTNAME']}"
     job = f"{os.environ['JOB_NAME']}: {os.environ['JOB_ID']}:{os.environ['SGE_TASK_ID']}"
 
-    input_draw = parameters["input_draw"]
-    random_seed = parameters["random_seed"]
-    model_specification_file = parameters["model_specification_file"]
-    branch_config = parameters["branch_configuration"]
+    job_parameters = jobs.JobParameters(**job_parameters)
+
     logger.info(f"Launching new job {job} on {node}")
-    logger.info(
-        "Starting job: {}".format(
-            (input_draw, random_seed, model_specification_file, branch_config)
-        )
-    )
+    logger.info(f"Starting job: {job_parameters}")
 
     try:
-        np.random.seed([input_draw, random_seed])
-        worker_ = get_current_job().id
-
-        from vivarium.framework.engine import SimulationContext
-        from vivarium.framework.utilities import collapse_nested_dict
-
-        configuration = {}
-        run_key = {"input_draw": input_draw, "random_seed": random_seed}
-
-        if branch_config is not None:
-            configuration.update(dict(branch_config))
-            run_key.update(dict(branch_config))
-
-        input_data_config = {
-            "input_draw_number": input_draw,
-        }
-        if vct_globals.ARTIFACT_PATH_KEY in dict(branch_config).get(
```

Comment:
This is some baffling logic.  We've already put the branch_configuration in the configuration and if it contains the key input_data.artifact_path, we're going to overwrite it with the same thing again with this + line 156 (old code)